Gemfile

```diff
@@ -52,3 +52,7 @@ gem "rspec-rails", "~> 5.0"
 gem "devise", "~> 4.8"
 
 gem "react-rails", "~> 2.6"
+
+gem "bootstrap", "~> 5.1"
+
+gem 'rack-cors', :require => 'rack/cors'
```

Comment:
You don't need CORS for this project. It is all on the same origin - localhost:3000
You can remove this line of code and run bundle